app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] `current_user` should be defined in the controllers. You have defined `current_user` in the _ActiveRecord_. As a best practice, kindly ensure to define _current_user_ as action in _ApplicationController_ (`app/controllers/application_controller.rb`)

Reply:
This issue is well resolved! ✔️